msteams-platform/bots/how-to/conversations/subscribe-to-conversation-events.md

```diff
@@ -399,7 +399,20 @@ async def on_teams_channel_restored(
 
 ### Team members added
 
-The `teamMemberAdded` event is sent to your bot the first time it is added to a conversation. The event is sent to your bot every time a new user is added to a team or group chat where your bot is installed. The user information that is ID, is unique for your bot and can be cached for future use by your service, such as sending a message to a specific user.
+The `teamMemberAdded` event is sent to your bot in the following scenarios:
+
+1. When the bot, itself, is installed and added to a conversation
+
+   > In team context, the activity's conversation.id is set to the `id` of the channel selected by the user during app installation or the channel where the bot was installed from.
+
+2. When a user is added to a conversation where the bot is installed
+
+   > User ids received in the event payload are unique to the bot and can be cached for future use, such as directly messaging a user.
+
+To determine if the new member added was the bot itself or a user, check the `Activity` object of the `turnContext`. If the `id` field of the `MembersAdded` object is the same as the `id` field of the `Recipient` object, then the member added is the bot, else it's a user. The bot's `id` generally is `28:<MicrosoftAppId>`.
+
+> [!TIP]
+> It's recommended for bots to rely on the `InstallationUpdate` event to determine when it's added or removed from a conversation [installation update](#installation-update-event).
```

Comment:
The tag-on text at the end looks strange:

Would it be better to rephrase this as "We recommend using the [installation update](#installation-update-event) event to determine when your bot is added or removed from a conversation."

Reply:
Updated.